src/server/pfs/server/driver.go

```diff
@@ -122,7 +123,7 @@ func newDriver(env Env) (*driver, error) {
 	return d, nil
 }
 
-func (d *driver) createRepo(txnCtx *txncontext.TransactionContext, repo *pfs.Repo, description string, update bool) error {
+func (d *driver) createRepo(ctx context.Context, txnCtx *txncontext.TransactionContext, repo *pfs.Repo, description string, update bool) error {
```

Comment:
See my previous comment.  I don't think we need to add another context to all of these.  We should keep using the one from the `TransactionContext`